static/src/stylesheets/module/onward/_feedback.scss

```diff
@@ -0,0 +1,55 @@
+
+.feedback__form {
+    border: 0px;
+    outline: none;
+    display: none;
+}
+
+.feedback__form--selected {
+    display: block;
+}
+
+.feedback__element {
+    margin-bottom: 1.5rem;
+    margin-top: 1.5rem;
+}
+
+.feedback__entry {
+    border: 1px solid #dcdcdc;
+    padding: .75rem;
+    outline: none;
+    -webkit-appearance: none;
+    -moz-appearance: none;
```

Comment:
Appropriate vendor-prefixing should be taken care of automatically at build time, so we can remove these from the code.